### DISCUSSION TITLE
Customize Advance Settings fields?

### DISCUSSION THREAD

**Discussion by cj** [strapi/strapi#7705] [Legacy Topics]
Hi,

Is it possible to customize the "Advance Setting" fields using injectedComponents?

![CleanShot 2020-09-02 at 12 19 58](https://user-images.githubusercontent.com/1819/92015811-f7c27080-ed16-11ea-8f7e-32d5dbe155b6.png)

Cheers!

**Comment by derrickmehaffy** [comment]
Can you clarify what you are looking to change or add?

**Reply by cj** [reply]
@derrickmehaffy As you currently can't add custom collection types here:
![CleanShot 2020-09-02 at 12 30 11](https://user-images.githubusercontent.com/1819/92016567-25f48000-ed18-11ea-9fd5-b11c62727ea4.png)
I have extended the JSON field type with my own plugin that I want to render different components based on the type they choose under "ADVANCE SETTINGS", so I want to add a type dropdown that lets you select from `default`, `IMDB` or any other component I have added.

I hope that makes sense!

**Reply by derrickmehaffy** [reply]
Ah in that case I'm not sure, let me ping @soupette or @HichamELBSI to see if they have any suggestions.

**Reply by soupette** [reply]
@cj it's currently not possible and I can't provide an ETA on when it will be.
